edx_proctoring/api.py

```diff
@@ -787,7 +787,7 @@ def remove_exam_attempt(attempt_id):
         )
 
 
-def get_all_exams_for_course(course_id):
+def get_all_exams_for_course(course_id, active_only=False):
```

Comment:
We are already passing keyword argument in the models method. I think we can just pass simple parameter here. We are following the same structure elsewhere.